__test__/components/SignupFormContainer.test.js

```diff
@@ -81,7 +75,6 @@ describe('Sign up form container test', () => {
     wrapper.instance().handleSubmit(emailEvent);
     wrapper.setState({ data: {} });
     wrapper.instance().handleBlur(usernameEvent);
-    wrapper.instance().validate();
```

Comment:
Why this removal?

Reply:
Because `Validate` is no more a method of that class. It's now a helper function.